packages/commerce-sdk-react/src/hooks/ShopperOrders/mutation.ts

```diff
@@ -81,16 +83,36 @@ export const SHOPPER_ORDERS_NOT_IMPLEMENTED = [
     'UpdatePaymentInstrumentForOrder'
 ]
 
+type UseShopperOrdersMutationHeaders = NonNullable<
+    Argument<Client[ShopperOrdersMutationType]>
+>['headers']
+type UseShopperOrdersMutationArg = {
+    headers?: UseShopperOrdersMutationHeaders
+    rawResponse?: boolean
+}
+
 /**
  * A hook for performing mutations with the Shopper Orders API.
  */
-export function useShopperOrdersMutation<Action extends ShopperOrdersMutationType>(action: Action) {
+function useShopperOrdersMutation<Action extends ShopperOrdersMutationType>(
+    action: Action,
+    arg?: Omit<UseShopperOrdersMutationArg, 'rawResponse'> & {rawResponse?: false}
```

Comment:
The mutation hooks will receive only `action` initially. The `arg` won't be available as input until later on... until the user performs something in the UI (like clicking a button).

Reply:
Here in this line, I think we can ignore what was done elsewhere in the query hooks, and simply use `UseShopperOrdersMutationArg` type as it is. No need to `Omit` anything.